### ISSUE TITLE
Editor does not show text

### ISSUE THREAD

**Issue by zhaich** [olivierkes/manuskript#472]
So this seems to be related to issues #315 and #319, where the project file seems to be corrupted and does not display data correctly. I have syncthing installed between two Linux machines and have opened the .msk file on both simultaneously, _without knowing this was not supported_. On one of the machines, the editor no longer displays text, but instead a small box in the bottom left corner. When placed into Full Screen mode, the text appears however so the data is still present. The rest of the project displays fine. It is only the editor mode that does not work properly. The other machine still displays the project correctly in editor mode. 

The workaround found in #315 does not work for me. Running Manuskript from terminal results in this:

`$ manuskript`
`Debug: Web rendering engine used: QWebView`
`Running manuskript version 0.8.0.`
`Note: No translator found or loaded for locale en_US.`
`Loading: /home/nick/Sync/Fiction/The Forge of Infinity/FOIArc1.msk`
`Detected file format version: 1. Zip: True.`
`Project /home/nick/Sync/Fiction/The Forge of Infinity/FOIArc1.msk loaded.`
`Icon theme "NumixMsk" not found.`

Any ideas? 
![capture](https://user-images.githubusercontent.com/23005176/52300903-130ca480-2957-11e9-97e8-29ca5460afb8.png)

**Comment by gedakc** [comment]
The console log line that shows "_Icon theme "NumixMsk" not found._" is indicative a problem with the installation of Manuskript.

How did you install Manuskript on this computer?

**Comment by zhaich** [comment]
Manuskript was installed on both systems using the AUR (Arch User Repository). One of them works fine and the other does not which is what confuses me. I uninstalled and reinstalled the problematic system in question but received the same results. The Pkgbuild looked to be in order before install. I should mention that the editor on the faulty machine did display text until this suddenly happened, so I'm feeling this could have been due to opening the file simultaneously through a file sync service. Unfortunately the laptop battery died on me so I'll have to wait to see if exporting and making a new project makes any difference in another update.

**Comment by gedakc** [comment]
When the primary working version of Manuskript is operation again you might try saving the project again.

Then close the primary working version of Manuskript and then start Manuskript on the other computer and try to open the project.

Be sure to have only one copy of Manuskript running and editing the project at a single time.

**Comment by zhaich** [comment]
Both computers use Artix Linux (variant of Arch Linux). I have loaded the project on the working version of Manuskript and the editor worked fine. After saving, I closed this version then waited a few minutes before opening the version on the other computer to give syncthing a chance to sync the project file. After this, I opened the other version and the result is the same: small box in the bottom left and the scroll bar leading nowhere. However, full screen mode did work correctly on the latter version which is also shown in a screen capture below. 
![capture1](https://user-images.githubusercontent.com/23005176/52308801-4d347100-296c-11e9-842b-d544a6970f00.png)
![capture2](https://user-images.githubusercontent.com/23005176/52308811-4f96cb00-296c-11e9-8878-69b03d2c4130.png)
![capture3](https://user-images.githubusercontent.com/23005176/52308812-51608e80-296c-11e9-8810-b4b2b39b9b7d.png)

For what it's worth, I started a new project file and repeated the above steps and found no issues with the editor between both versions. So at least in principal, syncthing seems to be syncing the file correctly. I'm beginning to think that in the past, I opened the latter version right when the file was being synced between both computers, which resulted in this mess.

Unfortunately, it seems I might have to export the current project and re-import into a new project to salvage this unless another solution can be found.

**Comment by gedakc** [comment]
In an effort to get the project on both computers you might try one or both of the following:


- try disabling **Keep revisions** under **Edit -> Settings -> Revisions** and then save the project again.

- try enabling **Save to one single file** under **Edit -> Settings -> General** and save the project.  Exit and restart Manuskript, open the project again, and then disable **Save to one single file** and save the project again.

**Comment by zhaich** [comment]
I have disabled Keep revisions and performed the enabling/disabling Save to one single file routine on the good version. The bad version however shows no difference from the screen capture above. I have also used the source code version of Manuskript on the bad computer with the same result as well.

As for copying the .msk file, it is copied to the other computer automatically by a synchronization program installed on both computers when there is any change within a file on either computer. Therefore, when the .msk file is saved on one computer, the file is copied and overwrites the existing file on the other computer.

I exported the existing project on the good computer into an ePub file, then imported that file into a newly created project. The result was a crash which is seen below:

`$ manuskript`
`Debug: Web rendering engine used: QWebView`
`Running manuskript version 0.8.0.`
`Note: No translator found or loaded for locale en_US.`
`Loading: /home/nick/Sync/Fiction/The Forge of Infinity/Arc1.msk`
`Detected file format version: 1. Zip: True.`
`Project /home/nick/Sync/Fiction/The Forge of Infinity/Arc1.msk loaded.`
`Icon theme "NumixMsk" not found.`
`Traceback (most recent call last):`
`  File "/opt/manuskript/bin/../manuskript/ui/importers/importer.py", line 134, in selectFile`
`    self.preview()`
`  File "/opt/manuskript/bin/../manuskript/ui/importers/importer.py", line 240, in preview`
`    result = self.startImport(previewModel)`
`  File "/opt/manuskript/bin/../manuskript/ui/importers/importer.py", line 298, in startImport`
`    self.settingsWidget)`
`  File "/opt/manuskript/bin/../manuskript/importer/pandocImporters.py", line 23, in startImport`
`    formatTo = self.getSetting("formatTo").value().lower(`
`AttributeError: 'NoneType' object has no attribute 'value'`
`KCrash: crashing... crashRecursionCounter = 2`
`KCrash: Application Name = python3.7 path = /usr/bin pid = 9900`
`KCrash: Arguments: /usr/bin/python3.7`
`KCrash: Attempting to start /usr/lib/drkonqi from kdeinit`
`sock_file=/run/user/1000/kdeinit5__0`

`[1]+  Stopped                 manuskript`

So now there's a whole different issue here. 

It appears the Arch build of Manuskript is far more broken than other builds, given the other issues surrounding it with NumixMsk not installing, Issue #450 and so on. In that case, without a solution I am forced to migrate elsewhere, but I do thank you for your time and support to help fix this.
